src/main/java/frc/team2412/robot/subsystems/DrivebaseSubsystem.java

```diff
@@ -193,6 +207,14 @@ public Command driveJoystick(
 				});
 	}
 
+	public void simpleDrive(double forward, double strafe, Rotation2d rotation) {
+
+		ChassisSpeeds chassisSpeeds = new ChassisSpeeds(0, 0, 0);
```

Comment:
Delete unnecessary initialization to `ChassisSpeeds`.

Reply:
done